resources/assets/js/components/WordCounter/types.ts

```diff
@@ -0,0 +1,23 @@
+export interface WordCounterProps {
```

Comment:
These don't need to be in their own file.

Reply:
In fact, component props should _definitely_ be in the same file as their component.

Reply:
Okay!